src/app/pages/account/users/user-form/user-form.component.ts

```diff
@@ -116,6 +120,37 @@ export class UserFormComponent {
   readonly shellProvider = new SimpleAsyncComboboxProvider(this.shellOptions$);
   readonly groupProvider = new SimpleAsyncComboboxProvider(this.groupOptions$);
 
+  get homeCreateWarning(): string {
+    const homeCreate = this.form.get('home_create').value;
+    const home = this.form.get('home').value;
+    const homeMode = this.form.get('home_mode').value;
+    if (this.isNewUser) {
+      if (!homeCreate && home !== '/nonexistent') {
+        return this.translate.instant(
+          'With this configuration, Home Directory under path {path} will not be created.',
```

Comment:
Maybe:
"With this configuration, the existing directory {path} will be used a home directory without creating a new directory for the user."

Reply:
Thanks! I'll make the change.